src/instruments/src/EFB/Settings/Settings.tsx

```diff
@@ -45,6 +46,7 @@ export const Settings = () => {
         { alias: t('Settings.AircraftOptionsPinPrograms.Title'), name: 'Aircraft Options / Pin Programs', component: <AircraftOptionsPinProgramsPage /> },
         { alias: t('Settings.SimOptions.Title'), name: 'Sim Options', component: <SimOptionsPage /> },
         { alias: t('Settings.Realism.Title'), name: 'Realism', component: <RealismPage /> },
+        { alias: t('Settings.GsxOptions.Title'), name: 'GSX Options', component: <GsxOptionsPage /> },
```

Comment:
I'm not really thrilled about adding a whole page just for one 3rd party add-on. 
Is it possible to at least hide it when GSX is not installed (there is probably some LVAR indicating that it is installed)
Also maybe a new page "3rd Party Settings" (or something better)  would make more sense than a GSX only page.

Reply:
Sounds good to me, I'll do that

Reply:
resolved [c764914](https://github.com/flybywiresim/a32nx/pull/7695/commits/c764914ebbb62b43414416c946df0837204de948)